docs/pages/steps/jira_add_comment.md

```diff
@@ -10,10 +10,7 @@ folder: steps
 
```

Comment:
[modified](https://github.com/bhram999/jira-steps-plugin/blob/1f28420f0b30d6938f837cfb9a5fb6e7ed8bcdae/docs/index.md)the index page with a practical scenario 

changes for steps documentation are on the way